Source/Core/DiscIO/Volume.h

```diff
@@ -66,7 +66,11 @@ class Volume
   virtual Partition GetGamePartition() const { return PARTITION_NONE; }
   virtual std::optional<u32> GetPartitionType(const Partition& partition) const { return {}; }
   std::optional<u64> GetTitleID() const { return GetTitleID(GetGamePartition()); }
+  // https://gcc.gnu.org/bugzilla/show_bug.cgi?id=80635
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wmaybe-uninitialized"
```

Comment:
Sadly, those produce warnings on MSVC (which in turn gets promoted to an error). 

But what surprises me a bit more is that the warning isn't raised on `GetPartitionType` literally 3 lines before this (where the same happens, except using a `u32` instead of a `u64`).
Does it go away if you use `std::nullopt` instead?

Reply:
Thanks, didn't know about std::nullopt.